### ISSUE TITLE
Font rubicon-icon-font.ttf missing in 2.4.3

### ISSUE THREAD

**Issue by andidev** [GeekyAnts/NativeBase#1850]
## react-native, react and native-base version
    "native-base": "2.4.3",
    "react-native": "0.55.3",
    "react": "16.3.1",

I get the following error when running `react-native run-ios`
![skarmavbild 2018-04-30 kl 01 39 43](https://user-images.githubusercontent.com/571389/39412118-94c8f79e-4c17-11e8-9ffb-8431115522f9.png)

**Comment by sergchernata** [comment]
+1

**Comment by hpelitebook745G2** [comment]
are there any updates?
i've tried removing node_modules and doing npm install. i still got the error.
here's my versions:
`react-native-cli: 2.0.1`
`react-native: 0.47.2`
`"native-base": "^2.3.1",`

**Comment by hpelitebook745G2** [comment]
I still got unexplainable errors after upgrading so I solved this by manually downloading `rubicon-icon-font.ttf` and pasted it in `node_modules\native-base\Fonts\`

**Comment by sergchernata** [comment]
For the record, I ended up removing `node_modules` folder, dumping `watchman`  cache, dumping other caches and using the following versions:

```
"native-base": "^2.3.9",
"react": "16.3.1",
"react-native": "0.55.3",
```

Now everything seems to run smoothly.

**Comment by SupriyaKalghatgi** [comment]
Latest fixed on `master`
Release by EOD

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.4.4](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.4.4)

**Comment by tilakranjan** [comment]
Download the file and put in the required location. This fixed in my case.
File Location:
https://github.com/shoutem/school/blob/master/LunchSpotApp/android/app/src/main/assets/fonts/rubicon-icon-font.ttf

**Comment by g33ktony** [comment]
Same here on RN 0.60.5 and native-base 2.13.6

**Comment by khalidos32** [comment]
@g33ktony i have the same issue on AppCenter, so i decided to fix the version on 2.12.1.
2.12.1 works fine on my xCode.
